Add Pichler LG-350 airvent modbus templates

martindybal opened the pull request.

martindybal committed:
Add Pichler LG-350 airvent modbus templates

marian-blazicek reviewed (changes requested):
V modul read skripte su nejake nestandardne uvodzovky a MODBUSR funkcia potrebuje mat 3 parametre, chyba datovy typ.

marian-blazicek commented:
Este jedna drobnost, v novej verzii maju premenne moznost konverzie, maju potom krajsiu ikonku a mernu jednotku.
![image](https://user-images.githubusercontent.com/103591138/171806524-f6947b58-d934-40c3-8c25-8eb29de86834.png)

martindybal commented:
@marian-blazicek děkuji za review. 

1. V jakém read skriptu je chyba? 
1. Jakou konverzní hodnotu mám zvolit pro výkon jednotky (množství vzduchu m3/h)

marian-blazicek commented:
> @marian-blazicek děkuji za review.
> 
> 1. V jakém read skriptu je chyba?
- v modulovom

> 2. Jakou konverzní hodnotu mám zvolit pro výkon jednotky (množství vzduchu m3/h)
- zoznam konverzii sa postupne rozsiruje, ale takuto zatial nemame

martindybal committed:
Update Pichler LG-350 Airvent.xml

Odstranění čtení chyb
Nahrazení přepínače módů za Allow bypass. Po konzultaci s výrobcem, je to přesnější popis toho co to dělá
Přidání datových typů

martindybal commented:
@marian-blazicek upraveno.
